Blockzilla/SiriShortcuts.swift

```diff
@@ -53,18 +63,30 @@ class SiriShortcuts {
         INVoiceShortcutCenter.shared.getAllVoiceShortcuts { (voiceShortcuts, error) in
             DispatchQueue.main.async {
                 guard let voiceShortcuts = voiceShortcuts else { return }
-                let foundShortcut = voiceShortcuts.filter { (attempt) in
+                var foundShortcut = voiceShortcuts.filter { (attempt) in
                     attempt.shortcut.userActivity?.activityType == type.rawValue
                     }.first
+                if foundShortcut == nil {
```

Comment:
A short comment explaining this may be nice (i.e. the erase intent is handled differently because it's in the background)